.github/ISSUE_TEMPLATE/BUG.md

```diff
@@ -0,0 +1,34 @@
+---
+name: 🐛 Bug Report
+about: Did something not work?
+title: 'Bug: '
+labels: 'bug'
+---
+
+# 🐛 bug report
+
+## Packages affected
+
+- [ ] Sandpack-client
+- [ ] Sandpack-react
+
+## Description of the problem
+
+## How has this issue affected you? What are you trying to accomplish?
```

Comment:
The back to back questions here come across as almost passive aggressive? Maybe rephrase:

What were you doing when the problem occurred'?